### PR TITLE
ZH-trans: add cluster-interactive.html

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                To interact with the Terminal, please use the desktop/tablet version
+                -->
+                要与终端交互，请使用桌面/平板
+            
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/1" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-font="Roboto" data-katacoda-fontheader="Roboto Slab" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;"></div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">Continue to Module 2<span class="btn__next">›</span></a>
```
Comment (tengqm) [inline content/zh/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html]: Continue to Module 2

Comment (xichengliudui) [reply content/zh/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html]: 已修改。

**Comment by fleeto** [comment]
`[trans-bot:merged]`

Thank you @xichengliudui, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/602) had been updated.

**Pull request merged**